layers/+spacemacs/spacemacs-defaults/packages.el

```diff
@@ -209,8 +209,10 @@
      ediff-merge-split-window-function 'split-window-horizontally)
     :config
     ;; show org ediffs unfolded
-    (require 'outline)
-    (add-hook 'ediff-prepare-buffer-hook #'outline-show-all)
+    (add-hook 'ediff-prepare-buffer-hook
+              #'(lambda ()
+                  (when (featurep 'outline)
```

Comment:
Using `fboundp` instead of `featurep` would prevent the byte-compiler from complaining about `outline-show-all` possibly not being defined.